src/MetaCompilation.Analyzers/UnitTests/Verifiers/CodeRefactoringVerifier.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Simplification;
+using Microsoft.CodeAnalysis.Text;
+using Test.Utilities;
+using Xunit;
+
+namespace MetaCompilation.Analyzers.UnitTests
+{
+    public abstract class CodeRefactoringVerifier : DiagnosticVerifier
```

Comment:
@sharwell Does the new testing library have APIs for code refactoring testing?

Reply:
Not yet.